CHANGELOG.md

```diff
@@ -5,6 +5,16 @@ All notable changes to the Adapt authoring tool are documented in this file.
 **IMPORTANT**: For information on how to **correctly and safely** update your installation, please consult **INSTALL.md**.<br/>
 _Note that we adhere to the [semantic versioning](http://semver.org/) scheme for release numbering._
 
+## [0.8.0] - 2019-05-01
```

Comment:
Please bump to today's date.

Reply:
no way

Reply:
@tomgreenfield want me to update this to today's date now?